bigbluebutton-html5/imports/ui/components/whiteboard/component.jsx

```diff
@@ -445,6 +446,13 @@ const Whiteboard = React.memo(function Whiteboard(props) {
     setTldrawAPI(editor);
 
     editor?.user?.updateUserPreferences({ locale: language });
+    
+    if (darkTheme) {
+      // force darkReader to reload to fix some cases
+      // where tldraw dark theme isn't applied
+      AppService.setDarkTheme(false);
+      AppService.setDarkTheme(true);
+    }

```

Comment:
From my tests this didn't seem to make a difference, did you notice it not applying sometimes?

Reply:
@Arthurk12 yeah I think it's better to remove it, not needed in this tldraw version